package.json

```diff
@@ -38,7 +39,6 @@
     "enzyme-adapter-react-16": "^1.9.1",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
-    "google-fonts-webpack-plugin": "^0.4.4",
```

Comment:
I am curious to know why we removed this.

Reply:
At some point the plugin became problematic and the error message was not comprehensive as such we decided to do away with it at the moment. Though we were thinking of referencig the CDN on index.html file